code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm

```diff
@@ -99,6 +100,38 @@
 	loot = list()
 	stat_attack = UNCONSCIOUS
 	robust_searching = 1
+	food_type = list(/obj/item/reagent_containers/food/snacks/customizable/salad/ashsalad, /obj/item/reagent_containers/food/snacks/customizable/soup/ashsoup)//use lavaland plants to feed the lavaland monster
+	tame_chance = 20
+	bonus_tame_chance = 10
+	var/saddled = FALSE
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/tamed(whomst)
+	var/mob/living/fren = whomst
+	friends = fren
+	faction = fren.faction.Copy()
+	..()
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/attackby(obj/item/O, mob/user, params)
+	if(istype(O, /obj/item/saddle) && !saddled)
+		if(tame && do_after(user,55,target=src))
+			user.visible_message("<span class='notice'>You manage to put [O] on [src], you can now ride him.</span>")
```

Comment:
make this refer to the goliath's gender pronoun instead of just assuming that the goliath will be a "him" (do xenobio gender change potions work on simplemobs?)

Reply:
would be funny if you used a genderswap potion on a goliath and it suddenly turned into a broodmother